cdm/zarr/src/test/java/thredds/inventory/zarr/TestMFileZip.java

```diff
@@ -0,0 +1,81 @@
+package thredds.inventory.zarr;
```

Comment:
I just realized the `MFileZip:MFileProvider` doesn't get used by TDS (so TDS can only have `MFileS3` or `MFileOS` implementations returned from `MFiles.create`). So I'm not sure if these tests are very useful/important-- maybe I should remove them.

Reply:
I think you should leave them, we're going to have to add MFileZip to the TDS if we want to support Zarr zip stores, which we probably do.